cluster/packages.sls

```diff
@@ -8,7 +8,7 @@
 {% endif %}
 
 {% if pattern_available == 0 %}
-{% set repo = salt['pkg.info_available']('patterns-ha-ha_sles')['patterns-ha-ha_sles']['repository'] %}
+{% set repo = salt['pkg.info_available']('patterns-ha-ha_sles', refresh=False)['patterns-ha-ha_sles']['repository'] %}
```

Comment:
@arbulu89 `nitpick`: perhaps we could add a comment before the `set` why we set `refresh` false

Reply:
this would perhaps future refactoring to add `refresh` thing